core-war/src/main/java/org/silverpeas/web/sharing/control/FileSharingSessionController.java

```diff
@@ -23,25 +23,49 @@
  */
 package org.silverpeas.web.sharing.control;
 
-import java.rmi.RemoteException;
-import java.util.ArrayList;
-import java.util.Date;
-import java.util.List;
-
-import org.silverpeas.core.sharing.services.SharingServiceProvider;
-import org.silverpeas.core.web.sharing.bean.SharingNotificationVO;
-import org.silverpeas.core.web.sharing.notification.FileSharingUserNotification;
-
+import org.apache.commons.lang3.tuple.Pair;
+import org.silverpeas.core.admin.PaginationPage;
+import org.silverpeas.core.admin.user.model.UserDetail;
+import org.silverpeas.core.sharing.model.DownloadDetail.QUERY_ORDER_BY;
 import org.silverpeas.core.sharing.model.Ticket;
+import org.silverpeas.core.sharing.services.SharingServiceProvider;
 import org.silverpeas.core.sharing.services.SharingTicketService;
+import org.silverpeas.core.util.SilverpeasList;
 import org.silverpeas.core.util.StringUtil;
 import org.silverpeas.core.web.mvc.controller.AbstractComponentSessionController;
 import org.silverpeas.core.web.mvc.controller.ComponentContext;
 import org.silverpeas.core.web.mvc.controller.MainSessionController;
-import org.silverpeas.core.admin.user.model.UserDetail;
+import org.silverpeas.core.web.sharing.bean.SharingNotificationVO;
+import org.silverpeas.core.web.sharing.notification.FileSharingUserNotification;
+
+import java.rmi.RemoteException;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.silverpeas.core.sharing.model.DownloadDetail.QUERY_ORDER_BY.*;
+import static org.silverpeas.core.sharing.model.Ticket.QUERY_ORDER_BY.*;
 
 public class FileSharingSessionController extends AbstractComponentSessionController {
 
+  public static final Map<Integer, Pair<Ticket.QUERY_ORDER_BY, Ticket.QUERY_ORDER_BY>>
+      TICKET_ORDER_BIES = new HashMap<>();
+  private static final int TICKET_CREATION_DATE_INDEX = 1;
+  private static final int TICKET_END_DATE_INDEX = 4;
+  private static final int TICKET_NBACCESS_DATE_INDEX = 5;
+  private static final int DEFAULT_TICKET_PAGINATION_SIZE = 25;
+  private PaginationPage ticketPagination;
+  private Ticket.QUERY_ORDER_BY ticketOrderBy;
+
+  public static final Map<Integer, Pair<QUERY_ORDER_BY, QUERY_ORDER_BY>> DOWNLOAD_ORDER_BIES =
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this member "protected". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS2386)